pecos/core/xmc/inference.hpp

```diff
@@ -41,6 +41,7 @@
 
 #define DEFAULT_LAYER_TYPE LAYER_TYPE_BINARY_SEARCH_CHUNKED
 
+using namespace robin_hood;
```

Comment:
This is risky. Please consider use something similar to [code](https://github.com/amzn/pecos/blob/mainline/pecos/core/utils/tfidf.hpp#L56).

Reply:
Line has been removed, instead _using robin_hood::unordered_set_ and _using robin_hood::unordered_map_ have been added within namespace pecos.